src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -32,7 +33,7 @@ internal static class OpsConfigLoader
                 return (default, default, default);
             }
 
-            var opsConfig = LoadOpsConfig(errors, package, repository.Path);
+            var opsConfig = LoadOpsConfig(errors, package, package.BasePath, Path.GetRelativePath(package.BasePath, repository.Path));
```

Comment:
Would it be enough to just input `workingDirectory` as `Path.GetRelativePath(package.BasePath, repository.Path)`?

Reply:
Then the parameter name `workingDirectory` can be really confusing.

---
In reply to: [562356342](https://github.com/dotnet/docfx/pull/7018#discussion_r562356342) [](ancestors = 562356342)

Reply:
Fixed.